.quickstart/quickstart.yml

```diff
@@ -0,0 +1,11 @@
+database_key: aws_cloud_cost_database
+schema_key: aws_cloud_cost_schema
+table_key: aws_cloud_cost_report_identifier
```

Comment:
You can remove this attribute and stick to our default structure because we are not implementing a generic solution to support this for all packages.

Reply:
Thanks for that note @fivetran-milanpavlov just removed in latest commit.